### ISSUE TITLE
Speedup Check of "best available dns-servers"

### ISSUE THREAD

**Issue by mwiora** [mwiora/namebench#4]
currently "Include best available regional DNS services" seems to be not really well working.

**Comment by rbrito** [comment]
No, that's a very good feature. We just actually need to check/ask users which regional servers work for them and keep the list updated.
